R/est.incidence.by.R

```diff
@@ -142,90 +158,80 @@ est.incidence.by <- function(
+      require(serocalculator) # note - this gets out of sync
+      # when using load_all() in development
       require(dplyr)
     })
 
-    {
-      fits <- parallel::parLapplyLB(
-        cl = cl,
-        X = stratumDataList,
-        fun = function(x) {
-          do.call(
-            what = est.incidence,
-            args = c(
-              x,
-              list(
-                lambda_start = lambda_start,
-                antigen_isos = antigen_isos,
-                build_graph = build_graph,
-                print_graph = FALSE,
-                verbose = FALSE,
-                ...
-              )
+    time <- system.time(fits <- parallel::parLapplyLB(
+      cl = cl,
+      X = stratumDataList,
+      fun = function(x) {
+        do.call(
+          what = est.incidence,
+          args = c(
+            x,
+            list(
+              lambda_start = lambda_start,
+              antigen_isos = antigen_isos,
+              build_graph = build_graph,
+              print_graph = FALSE,
+              verbose = FALSE,
+              ...
             )
           )
-        }
-      )
-    } %>% system.time() -> time
+        )
+      }
+    ))
 
     if (verbose) {
       message("Elapsed time for parallelized code: ")
       print(time)
     }
   } else {
-    # fits <- lapply(
-    #   X = stratumDataList,
-    #   FUN = function(x) est.incidence(dataList = x, verbose = verbose, ...))
-
     fits <- list()
 
-    { # time progress
-
-      for (cur_stratum in names(stratumDataList))
-      {
-        cur_stratum_vars <-
-          strata_table %>%
-          dplyr::filter(.data$Stratum == cur_stratum)
-
-        if (verbose) {
-          message("starting new stratum: ", cur_stratum)
-          print(cur_stratum_vars)
-        }
-
-        fits[[cur_stratum]] <-
-          do.call(
-            what = est.incidence,
-            args = c(
-              stratumDataList[[cur_stratum]],
-              list(
-                lambda_start = lambda_start,
-                antigen_isos = antigen_isos,
-                build_graph = build_graph,
-                print_graph = print_graph,
-                verbose = verbose,
-                ...
-              )
-            )
-          )
+    # Time progress
+    for (cur_stratum in names(stratumDataList)) {
+      cur_stratum_vars <- strata_table %>%
+        dplyr::filter(.data$Stratum == cur_stratum)
+
+      if (verbose) {
+        message("starting new stratum: ", cur_stratum)
+        print(cur_stratum_vars)
       }
-    } %>% system.time() -> time
+
+      fits[[cur_stratum]] <- do.call(
+        what = est.incidence,
+        args = c(
+          stratumDataList[[cur_stratum]],
+          list(
+            lambda_start = lambda_start,
+            antigen_isos = antigen_isos,
+            build_graph = build_graph,
+            print_graph = print_graph,
+            verbose = verbose,
+            ...
+          )
+        )
+      )
```

Comment:
Changing how the code operates to avoid `->` assignment which bring up a warning on linting. Assignment should right to left.